docs/site/migration/components/models.md

```diff
@@ -6,8 +6,63 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/migration-extensions-models.html
 ---
 
-{% include note.html content="
-This is a placeholder page, the task of adding content is tracked by the
-following GitHub issue:
-[loopback-next#3955](https://github.com/strongloop/loopback-next/issues/3955)
-" %}
+Please review the content in [Migrating models](../models/overview.md) first.
+Going forward, we assume that you are already familiar with the differences
+between LoopBack 3 Models and LoopBack 4 Entities & Repositories, and understand
+how to migrate model-related functionality from a LoopBack 3 application to
+LoopBack 4.
+
+## Import LoopBack 3 models
+
+The first step step is to import LoopBack 3 model definitions into your LoopBack
+4 component. This will convert LB3 model JSON files into LB4 TypeScript classes,
+as explained in
+[Import Model definition](../models/core.md#import-model-definition) and
+[Importing models from LoopBack 3 projects](../../Importing-LB3-models.md).
+
+1. Create a small LB3 app that is using your component.
+
+2. In your LB4 extension project, run `lb4 import-lb3-models` to import model(s)
```

Comment:
@bajtos I remember `lb4 import-lb3-models` takes in a source directory as argument/option, could you specify the dir here? Should it be the path to lb3 app or the component?

Reply:
Good idea, I'll clarify that the path should point to the lb3 app (that's why we need to create it).